cypress/support/pages/course/pageImportCourseModal.js

```diff
@@ -0,0 +1,33 @@
+"use strict"
+
+class ImportCourseModal {
+
+	static #importShareCourseDialog = '[data-testid="import-modal"]'
+	static #importShareCourseToolsInfo = '[data-testid="import-modal-external-tools-info"]'
+	static #importShareCourseNameInput = '[data-testid="import-modal-name-input"]'
+	static #dialogConfirmButton = '[data-testid="dialog-confirm"]';
+
+	seeImportShareCourseDialogBox() {
+		cy.get(ImportCourseModal.#importShareCourseDialog).should("exist");
+	}
+
+	seeImportShareCourseToolsInfo() {
+		cy.get(ImportCourseModal.#importShareCourseToolsInfo).should("exist");
+	}
+
+	enterCourseNameForImportCourse(input) {
+		cy.get(ImportCourseModal.#importShareCourseNameInput).should("exist");
+		cy.get(ImportCourseModal.#importShareCourseNameInput).clear();
+		cy.get(ImportCourseModal.#importShareCourseNameInput).type(input);
+	}
+
+	clickOnConfirmButtonInDialog() {
```

Comment:
maybe extend the name in which dialog the button is
e.g. clickOnConfirmButtonInImportShareCourseDialog